flambe/task/legacy.py

```diff
@@ -1,4 +1,5 @@
 import math
```

Comment:
Any reason to keep the existing code? If this is a backwards incompatible change, lets throw the current implementation so that we don't maintain it.

Reply:
Like we discussed offline, we keep the old code for allowing people to smoothly transition